src/main/java/seedu/address/logic/commands/FilterDentistCommand.java

```diff
@@ -15,8 +19,15 @@ public class FilterDentistCommand extends Command {
     public static final String COMMAND_WORD = "filter-dentist";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters the list of dentists by a specific criteria.\n"
-        + "Parameters: a/ (attribute) k/ (keyword)\n"
-        + "Example: " + COMMAND_WORD + " a/ gender k/ Male";
+        + "Parameters: a/(attribute) k/(keyword)\n"
```

Comment:
maybe can follow the format for other classes?
e.g. "Parameter: a/ATTRIBUTE k/KEYWORD\n"